src/P3_size_distribution.jl

```diff
@@ -96,11 +96,13 @@ function get_ice_bound(p3::PSP3, λ::FT, tolerance::FT) where {FT}
 end
 
 """
-    L_(p3, ρ, F_r, λ, μ, D_min, D_max)
+    L_(p3, ρ, F_rim, F_liq, λ, μ, D_min, D_max)
 
  - p3 - a struct with P3 scheme parameters
  - ρ - bulk ice density (ρ_i for small ice, ρ_g for graupel) [kg/m^3]
- - F_r - rime mass fraction [L_rim/L_ice]
+ - F_rim - rime mass fraction (L_rim / (L_ice - L_liq)) [-]
+ - F_liq - liquid fraction (L_liq / L_ice) [-]:
```

Comment:
It seems to me that the notation is a little confusing between `L_ice` and `L_liq`. Maybe it would be better to have `L_tot = L_ice + L_liq`?

Reply:
Or `L_p3_tot`? I'm thinking out loud here, not sure.

Reply:
Ah yeah, maybe we can finalize the best way this afternoon. I think `L_tot = L_ice + L_liq` is what I was going to do originally, but if we make `L_tot = L_ice + L_liq`, when we switch into a model like KiD, I was worried that would be confusing when we're talking about `q_tot = q_ice + q_liq + q_rai + ...` where `q_liq` is the cloud liquid mass and not the mass of liquid on ice particles like P3's `L_liq` is. In the paper they have `L_ice,tot = L_ice + L_liq` but that seemed like too much, so I went with `L_ice = L_rim + L_dep + L_liq`. `L_p3_tot` could be a good solution because that would notation would transfer well to models where we have `L_tot = L_p3_tot + L_other`. I'll change to `L_tot_p3` this morning since I like the way that sounds more than `L_p3_tot` 😄, and we can check in again this afternoon.